lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3627,19 +3633,31 @@ swift::ModuleDecl *SwiftASTContext::GetModule(const SourceModule &module,
   auto import_diags = getScopedDiagnosticConsumer();
 
   // Report progress on module importing by using a callback function in
-  // swift::ASTContext
+  // swift::ASTContext.
   Progress progress("Importing Swift modules");
-  ast->SetPreModuleImportCallback([&progress](llvm::StringRef module_name,
-                                              bool is_overlay) {
-    progress.Increment(
-        1, (is_overlay ? module_name.str() + " (overlay)" : module_name.str()));
-  });
+  ast->SetPreModuleImportCallback(
+      [&progress](llvm::StringRef module_name,
+                  swift::ASTContext::ModuleImportKind kind) {
+        switch (kind) {
+        case swift::ASTContext::Module:
+          progress.Increment(1, module_name.str());
+          break;
+        case swift::ASTContext::Overlay:
+          progress.Increment(1, module_name.str() + " (overlay)");
+          break;
+        case swift::ASTContext::BridgingHeader:
+          progress.Increment(1,
+                             "Compiling bridging header: " + module_name.str());
+          break;
```

Comment:
In the `swift::ASTContext::Module` case, we only put the name of the module, if it's an overlay, we show both the name and `(overlay)` annotation, but if it's a bridging header, we now show `Compiling bridging header:` with the module name.

I don't any preference but let's stay consistent across all the `Increment` `updated_detail` string.